<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Keytest</class>
 <widget class="QWidget" name="Keytest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Keytest</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="mainLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="pbt_up">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>V+</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pbt_down">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>V-</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
<!-- <layoutdefault spacing="6" margin="11"/> 
 <resources/>
 <connections/>
 -->
</ui>
